generators/spring-data-relational/templates/src/main/java/_package_/_entityPackage_/domain/_persistClass_.java.jhi.spring_data_persistable.ejs

```diff
@@ -36,7 +36,10 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 <&_ } -&>
 
 <&_ if (fragment.classAdditionalFieldsSection) { -&>
+    @org.springframework.data.annotation.Transient
+<%_ if (!reactive) { _%>
```

Comment:
```suggestion
<%_ if (!reactive && databaseTypeSql) { _%>
```

Jakarta's Transient should be used in hibernate only?
Others databases should be compatible with Spring Data's Transient

Reply:
Nevermind isPersistable is Sql only.